ko/csql.rst

```diff
@@ -824,3 +824,26 @@ CSQL 인터프리터에서 작업 중인 데이터베이스 이름 및 호스트
     SINGLE IS OFF
     csql> ;singleline
     SINGLE IS OFF
+
+**새로운 csql 세션 연결(;Connect)**
```

Comment:
```suggestion
** csql 세션 전환 (;Connect)**
```

Reply:
Could we consider this because If switching is more appropriate, I think the name of ;Connect also should be ;Switch
(In the code level, it also disconnects current session and connect new one.)
@kisoo-han 
How could you think of this?

Reply:
@eido5 
I think you don't need to change session command, because ;connect is familiar to the user.

Reply:
@mhoh3963 
Could you tell me why words switching(전환) is better?
To me, from [CBRD-24620](http://jira.cubrid.org/browse/CBRD-24620) I used only 'connect' words so I am confused.
So, please help me.

(I knew command name is set to 'connect' as we compromised when we set specification.)

Reply:
@eido5 
The phrase "새로운 csql 세션 연결" gives the user the impression that one more new csql session has been added.
So, I recommend you changing to "전환".

Reply:
@mhoh3963 
Thank you. Now I can understand your intention. I will apply it.